model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -28,6 +28,7 @@ import org.ossreviewtoolkit.model.config.CopyrightGarbage
 import org.ossreviewtoolkit.model.config.LicenseFilePatterns
 import org.ossreviewtoolkit.model.config.PathExclude
 import org.ossreviewtoolkit.model.utils.PathLicenseMatcher
+import org.ossreviewtoolkit.utils.common.FileMatcher
```

Comment:
only maybe related: Are we actually missing some kind of `distinct()` call on `relativeFilePaths` in ...?
```
fun getApplicableLicenseFilesForDirectories(
        relativeFilePaths: Collection<String>,
        directories: Collection<String>
)
```